### PR TITLE
fix for interpolate-4

### PR DESCRIPTION
**Pull request by chenhu-wang**
do not change index table when execute each time

### CODE REVIEW

**Comment by chenhu-wang** [comment]
@dmitry-gorokhov , could you please review this fix?

**Comment by dmitry-gorokhov** [comment]
> @dmitry-gorokhov , could you please review this fix?

This PR fixes bug in the implementation, am I right? If so, is it possible to cover this case with test?

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp [hunk 1]
```diff
@@ -1992,7 +1992,6 @@ void MKLDNNInterpolateNode::linearInterpolation(const uint8_t *in_ptr_, uint8_t
     });
 }
 
-
```
Comment (antonvor) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp:1995]: empty line

**Comment by antonvor** [comment]
CheckCPUImpl(executableNetwork, "interpolate", inFmts, outFmts, selectedType);

I just noticed that you have `interpolate` here instead `Interpolate`. This check is most likely not valid. May I ask you to check this?

**Comment by chenhu-wang** [comment]
> > @dmitry-gorokhov , could you please review this fix?
> 
> This PR fixes bug in the implementation, am I right? If so, is it possible to cover this case with test?

Yes, the first time infer is OK, but second time infer will be wrong I believe. The index is pre-built,  each time in execute(), the index is updated one time by in-place manner. we should not change original index in each infer. I am thinking how to add test for more than one time infer.

**Comment by chenhu-wang** [comment]
> CheckCPUImpl(executableNetwork, "interpolate", inFmts, outFmts, selectedType);
> 
> I just noticed that you have `interpolate` here instead `Interpolate`. This check is most likely not valid. May I ask you to check this?

Fixed. Thank you! could you please check [this change](https://github.com/openvinotoolkit/openvino/pull/2152/files#diff-74dec2de76fb5f0e61925423a5b71f6aR90) as well when check more than one input memory format.

**Comment by chenhu-wang** [comment]
the change is combined as part of https://github.com/openvinotoolkit/openvino/pull/2281

**Pull request closed without merging**